pkgs/applications/audio/spotify-spicified/default.nix

```diff
@@ -0,0 +1,118 @@
+{ lib
+, spotify-unwrapped
+, spicetify-cli
+, spicetify-themes
+
+  # Spicetify settings
+, theme ? ""
+, colorScheme ? ""
+, thirdParyThemes ? {}
+, thirdParyExtensions ? {}
+, thirdParyCustomApps ? {}
+, enabledExtensions ? []
+, enabledCustomApps ? []
+, spotifyLaunchFlags ? ""
+, injectCss ? false
+, replaceColors ? false
+, overwriteAssets ? false
+, disableSentry ? true
+, disableUiLogging ? true
+, removeRtlRule ? true
+, exposeApis ? true
+, disableUpgradeCheck ? true
+, fastUserSwitching ? false
+, visualizationHighFramerate ? false
+, radio ? false
+, songPage ? false
+, experimentalFeatures ? false
+, home ? false
+, lyricAlwaysShow ? false
+, lyricForceNoSync ? false
+}:
+
+with lib;
+
+let
+  # Helper functions
+  pipeConcat = lists.foldr (a: b: a + "|" + b) "";
+  lineBreakConcat = foldr (a: b: a + "\n" + b) "";
```

Comment:
lineBreakConcat =  list: lib.concatStringsSep "\n" list; You would have to add line `51`, `52`, `53` and `56` to a list.